packages/edit-site/src/components/sidebar-navigation-screen/index.js

```diff
@@ -39,7 +39,19 @@ export default function SidebarNavigationScreen( {
 							) }
 						/>
 					) : (
-						<div className="edit-site-sidebar-navigation-screen__icon-placeholder" />
+						<NavigatorBackButton
+							className="edit-site-sidebar-navigation-screen__back"
+							icon={ isRTL() ? chevronRight : chevronLeft }
+							aria-label={ __( 'Navigate to the Dashboard' ) }
+							href="../wp-admin"
```

Comment:
This should be using the "dashbordURL" config we have.

Reply:
Yep, for sure, this was just a temporary hack while I was trying to get the `href` working, will update it when I change it to a normal button.